aie_runtime_lib/AIE2/tanh.cpp

```diff
@@ -0,0 +1,148 @@
+//===---  tanh.cpp - tanh loopup tables ---===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2023 Xilinx Inc.
+//
+//
+//===----------------------------------------------------------------------===//
+// These are tanh lookup tables for bfloat16 type
+//===----------------------------------------------------------------------===//
+
+// Divides into 32 segments between [-4,4], bank size: (32*2*2*4)*2=1k, one
+// lut=512B
+float chess_storage(% chess_alignof(v32int8)) tanh_lut_ab[2048] = {
```

Comment:
The initializer list for tanh_lut_ab has 128 elements. Why is the size of tanh_lut_ab 2048?